ktor-client/ktor-client-curl/posix/src/io/ktor/client/engine/curl/CurlClientEngineConfig.kt

```diff
@@ -1,9 +1,20 @@
 /*
- * Copyright 2014-2019 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ * Copyright 2014-2020 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
  */
 
 package io.ktor.client.engine.curl
 
 import io.ktor.client.engine.*
 
-public class CurlClientEngineConfig : HttpClientEngineConfig()
+public class CurlClientEngineConfig : HttpClientEngineConfig() {
+    /**
+     * Forces proxy tunneling by setting CURLOPT_HTTPPROXYTUNNEL
+     */
+    internal var forceProxyTunneling: Boolean = false
+
+    /**
+     * Enable TLS host and certificate verification by setting options
+     * CURLOPT_SSL_VERIFYPEER and CURLOPT_SSL_VERIFYHOST
+     */
+    internal var sslVerify: Boolean = true
```

Comment:
Are we going to make these fields public in the future? If so it would be great if we make parallel PR in `1.5.0`

Reply:
Yes, I think we do. Will do that.

Reply:
Created https://youtrack.jetbrains.com/issue/KTOR-1093 to not forget